python/pyspark/sql/functions.py

```diff
@@ -11738,15 +11738,29 @@ def create_map(
 
     >>> from pyspark.sql import functions as sf
     >>> df = spark.createDataFrame([("Alice", 2, "female"),
-    ...                             ("Bob", 5, "male")], ("name", "age", "gender"))
+    ...     ("Bob", 5, "male")], ("name", "age", "gender"))
```

Comment:
Why is there a need to modify these two lines?

Reply:
Because our Python style uses 4 spaces for indentation, I suggest maintaining consistent indentation in the same code snippet (Existing examples and newly added examples.)

Reply:
If this is a rule, I suggest fixing it in a separate pr, as there are similar cases to be addressed.